spread-shellcheck

```diff
@@ -83,7 +86,10 @@ class ShellcheckFailures(Exception):
         return len(self.failures)
 
     def contains(self, other):
```

Comment:
this name was and is a bit confusing I think

Reply:
Now that the method has changed a little bit maybe `contained` or just `intersection` would work?

Reply:
probably obvious is best, 'intersection'